src/components/Portal.svelte

```diff
@@ -0,0 +1,25 @@
+<script>
+  /* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Do we need the license header at the top of embedded script like this? IIRC the license agreement should be in a `LICENSE` file in the repo and also at the top of each file, but I'm not sure that we need to for embedded code like this (I don't recall seeing it before anyway)

Reply:
I think it should be at the top of each file (regardless of the purpose), where possible. AFAIK this is what m-c and other repos do, but I believe that we might to check with legal if we need to change that (I have no strong opinion!)

Reply:
I should also note that this is what the linter automatically adds – in the script block. Can move this above it since the notice pertains to the entire file.

Reply:
I filed #184 to track fixing this, and I won't address it in this PR since it will technically break CI. Marking this and the other ones as resolved wit hthis in mind.